tools/deployment/package-helm/templates/webui-deployment.yaml

```
@@ -55,6 +55,8 @@ spec:
               value: "4000"
             - name: "RATE_LIMIT"
               value: {{ .Values.clpConfig.webui.rate_limit | quote }}
+            - name: "SEARCH_MAX_NUM_RESULTS"
+              value: {{ .Values.clpConfig.webui.max_num_results | default 1000 | quote }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)